packages/skeleton-react/src/routes/components/progress.tsx

```diff
@@ -48,9 +48,9 @@ export function Component() {
 				<p>bg-primary-500 (default)</p>
 				<Progress value={value} max={max} />
 				<p>bg-secondary-500</p>
-				<Progress value={value} max={max} meterClasses="bg-secondary-500" />
+				<Progress value={value} max={max} meterBg="bg-secondary-500" />
```

Comment:
FYI, no change need here, but we should be using Tailwind convention for style prop names. So `bg` is correct , but I believe we've strayed from this in other components and fell back to `background`. Just a heads up! This is something I hope to correct soon.

Reply:
`meterClasses` does exist as a prop on Progress, but it is applied after `meterBg`

```html
<div class="{meterBase} {meterBg} {meterRounded} {meterTransition} {rxIndeterminate} {meterClasses}" style:width={fillPercentage}></div>
```

The default is e.g.  `bg-surface-500` if a custom bg e.g. `bg-secondary-500` is applied to `meterClasses`, then the resulting string would be:
```html
<div class="bg-surface-500 bg-secondary-500" style:width={fillPercentage}></div>
```

The end example then just shows `bg-surface-500`.

I just changed this to have a working example in the sandbox.

Reply:
Sorry I was speaking specifically to the prop name - no change needed there. The change to where the color prop is provided is valuable and should remain!